src/CodiceFiscale/Subject.php

```diff
@@ -89,7 +94,7 @@ public function setSurname($surname)
     /**
      * Get the birthDate.
      *
-     * @return Returns the birthDate.
+     * @return DateTime Returns the birthDate.
```

Comment:
You could remove Returns here and use an uppercase **T** for the initial **the**.